src/ai/backend/manager/api/session.py

```diff
@@ -2807,4 +2832,7 @@ def create_app(
     cors.add(app.router.add_route("POST", "/{session_name}/commit", commit_session))
     cors.add(app.router.add_route("GET", "/{session_name}/commit", get_commit_status))
     cors.add(app.router.add_route("GET", "/{session_name}/abusing-report", get_abusing_report))
+    cors.add(
+        app.router.add_route("POST", "/{session_name}/sync-agent-registry", sync_agent_registry)
```

Comment:
This should not take `session_name` parameter. Let's use `/_/sync-agent-registry`.